src/util.cpp

```diff
@@ -213,7 +213,6 @@ void PrintExceptionContinue(const std::exception* pex, const char* pszThread)
 {
     std::string message = FormatException(pex, pszThread);
     LogPrintf("\n\n************************\n%s\n", message);
-    fprintf(stderr, "\n\n************************\n%s\n", message.c_str());
```

Comment:
I'm confused; we do `LogPrintf` in many other places and I haven't previously seen those messages be visible on the metrics screen. I guess that could be something to do with the exception being a multi-line message?

Reply:
`LogPrintf` only goes to `debug.log`, not to the metrics screen. The metrics screen shows alerts / warnings via that subsystem, but here the direct stderr printing is what shows it up (with the logic being that an exception reaching this method is something the operator should be loudly warned about).